source/_includes/puppet-collections/_puppet_collection_1_contents.md

```diff
@@ -0,0 +1,6 @@
+Package                              | Contents
+-------------------------------------|----------------------------------------------
+[`puppet-agent`](./about_agent.html) | Puppet, [Facter](/facter/), [Hiera](/hiera/), [MCollective](/mcollective), `pxp-agent`, root certificates, and prerequisites like [Ruby](https://www.ruby-lang.org/) and [Augeas](http://augeas.net/)
+`puppetserver`                       | [Puppet Server](/puppetserver/); depends on `puppet-agent`
+`puppetdb`                           | [PuppetDB](/puppetdb/)
+`puppetdb&#8209;termini`                   | Plugins to let [Puppet Server talk to PuppetDB](/puppetdb/latest/connect_puppet_master.html)
```

Comment:
It's a non-breaking hyphen entity. With a normal hyphen, `puppet-termini` breaks into two overlapping lines because the right column expands to all possible space and the preformatted styling is taller than our line-height.